src/slam_mapper.cpp

```diff
@@ -85,217 +85,209 @@ karto::Pose2 SMapper::toKartoPose(const tf2::Transform & pose) const
 }
 
 /*****************************************************************************/
-void SMapper::configure(const rclcpp::Node::SharedPtr & node)
+void SMapper::configure(const rclcpp::Node::SharedPtr &node)
```

Comment:
Revert, this breaks ROS2 linters

Reply:
Sorry for that, applied your suggested change.

A general question concerning that: Would it be preferred to edit the original commits to keep the history clean instead of adding additional commits for requested changes like this? If so I am happy to modify it accordingly.

When running `ament_uncrustify` on this source file I also get some line-break divergences (see log below).
Should this be fixed too or am I using the wrong linter/settings here?
Again, sorry if that should be obvious.

**ament_uncrustify (src/slam_mapper.cpp):** 
```
ament_uncrustify slam_toolbox/src/slam_mapper.cpp 
Code style divergence in file 'slam_toolbox/src/slam_mapper.cpp':

--- slam_toolbox/src/slam_mapper.cpp
+++ slam_toolbox/src/slam_mapper.cpp.uncrustify
@@ -60,2 +60,3 @@
-  return karto::OccupancyGrid::CreateFromScans(mapper_->GetAllProcessedScans(),
-           resolution);
+  return karto::OccupancyGrid::CreateFromScans(
+    mapper_->GetAllProcessedScans(),
+    resolution);
@@ -88 +89 @@
-void SMapper::configure(const rclcpp::Node::SharedPtr &node)
+void SMapper::configure(const rclcpp::Node::SharedPtr & node)
@@ -171 +172,3 @@
-    node->declare_parameter("loop_match_maximum_variance_coarse", loop_match_maximum_variance_coarse);
+    node->declare_parameter(
+      "loop_match_maximum_variance_coarse",
+      loop_match_maximum_variance_coarse);
@@ -178 +181,3 @@
-    node->declare_parameter("loop_match_minimum_response_coarse", loop_match_minimum_response_coarse);
+    node->declare_parameter(
+      "loop_match_minimum_response_coarse",
+      loop_match_minimum_response_coarse);
@@ -193 +198,3 @@
-    node->declare_parameter("correlation_search_space_dimension", correlation_search_space_dimension);
+    node->declare_parameter(
+      "correlation_search_space_dimension",
+      correlation_search_space_dimension);
@@ -200 +207,3 @@
-    node->declare_parameter("correlation_search_space_resolution", correlation_search_space_resolution);
+    node->declare_parameter(
+      "correlation_search_space_resolution",
+      correlation_search_space_resolution);
@@ -207 +216,6 @@
-    node->declare_parameter("correlation_search_space_smear_deviation",
+    node->declare_parameter(
+      "correlation_search_space_smear_deviation",
+      correlation_search_space_smear_deviation);
+  }
+  node->get_parameter(
+    "correlation_search_space_smear_deviation",
@@ -209,2 +222,0 @@
-  }
-  node->get_parameter("correlation_search_space_smear_deviation", correlation_search_space_smear_deviation);

1 files with code style divergence
```